includes/class-coblocks-block-assets.php

```diff
@@ -464,6 +495,18 @@ public function frontend_scripts() {
 			);
 		}
 
+		// Services block.
+		if ( $this->is_page_gutenberg() || has_block( 'coblocks/services' ) ) {
```

Comment:
Rather than checking for the block then enqueueing this script, it needs to be added as a dependency of the Services block. So each time the services block renders, this script gets called.

Reply:
Okay, I better understand how these block scripts are being call. Not the most intuitive method, but it's clear you are following what already exists.

I was under the impression that block scripts were enqueued automatically, but I can see now it is a manual enqueue, in this method.